src/test/scala/au/com/titanclass/streams/telemetry/TracingJsonProtocolTest.scala

```diff
@@ -66,7 +62,7 @@ object TracingJsonProtocolTest extends TestSuite {
         .map { s =>
           val json = s.toJson
           val re =
-            """\{"baggage":\[\],"duration":.*,"logs":\[\{"time":0,"fields":\{\},"message":"hello-world"\},\{"time":0,"fields":\{"f1":"1","f0":"0"\}\}\],"operationName":"some-span","references":\[\],"spanId":.*,"start":.*,"tags":\{"sampler.type":"const","sampler.param":"true"\},"traceId":.*\}""".r
+            """\{"baggage":\[],"duration":.*,"logs":\[\{"time":0,"fields":\{},"message":"hello-world"},\{"time":0,"fields":\{"f0":"0","f1":"1"}}],"operationName":"some-span","references":\[],"spanId":.*,"start":.*,"tags":\{"sampler.type":"const","sampler.param":"true"},"traceId":.*}""".r
           assertMatch(re.findFirstIn(json.compactPrint)) {
```

Comment:
Its a bit crappy to make string comparisons given that differing dependencies will (have) affected the order. We're using regex here though, and so I don't want to over-complicate things.